src/Oro/Bundle/UserBundle/Controller/UserController.php

```diff
@@ -87,9 +88,9 @@ public function createAction()
      * @Template
      * @AclAncestor("pim_user_user_edit")
      */
-    public function updateAction(User $entity)
+    public function updateAction($id)
```

Comment:
in fact it can receive the Pim\Bundle\UserBundle\Entity\User and the PimEnterprise\Bundle\UserBundle\Entity\User which has additional methods. If I typehint here it will think it's a Pim user and will troncate the data